src/App.tsx

```diff
@@ -1,13 +1,29 @@
+import React, { useState } from "react";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 import Logo from "@assets/logo.png";
 import ActivitiesList from "@components/ActivitiesList/ActivitiesList";
 import { ACTIVITIES_MOCK } from "@components/ActivitiesList/mocks";
 import DataContainer from "@components/DataContainer/DataContainer";
 import HelloWorld from "@components/HelloWorld/HelloWorld";
-import { BrowserRouter, Route, Routes } from "react-router-dom";
+import AccountCreation from "@components/AccountCreation/AccountCreation";
+import AuthenticatedContent from "@components/AuthenticatedContent/AuthenticatedContent";
 
 import styles from "./App.module.css";
 
 export default function App() {
+  const [isAuthenticated, setIsAuthenticated] = useState(false);
+
+  const handleAccountCreate = (
+    username: string,
+    email: string,
+    password: string
+  ) => {
+    console.log(
+      `Account created: Username - ${username}, Email - ${email}, Password - ${password}`
+    );
+    setIsAuthenticated(true);
+  };
+
```

Comment:
nice!

Let's find out from backend what type of authentication data we can expect to receive from them, so we can make sure these checks will work with the api 👍